test/shared_test/utils.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "utils.hpp"
+#include <fstream>
+
+namespace ort_extensions {
+namespace test {
+std::vector<uint8_t> LoadBytesFromFile(const std::filesystem::path& filename) {
+  using namespace std;
+  ifstream ifs(filename, ios::binary | ios::ate);
```

Comment:
>ifs [](http://example.com/codeflow?start=11&length=3)

do we need to call ifs.close() before exit?

Reply:
Dtor closes the file. https://en.cppreference.com/w/cpp/io/basic_ifstream